buildSrc/src/main/java/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.kt

```diff
@@ -109,11 +109,13 @@ class FirebaseLibraryPlugin : BaseFirebaseLibraryPlugin() {
       from(project.zipTree("build/outputs/aar/${releaseAar}"))
       into(project.file("semver/current-version"))
     }
+
     project.tasks.register<Copy>("extractPreviousClasses") {
       dependsOn("copyPreviousArtifacts")
-
-      from(project.zipTree("semver/previous.aar"))
-      into(project.file("semver/previous-version"))
+      if (project.file("semver/previous.aar").exists()) {
```

Comment:
Keep in mind that if this file is added by any task- this will fail. This check will occur at configuration time, not execution time.